docker-compose.yml

```diff
@@ -1,15 +1,28 @@
-version: "2"
+version: "3.7"
 services:
+  nginx:
+    image: nginx:latest
+    container_name: ng01
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./src:/src
+      - ./config/nginx:/etc/nginx/conf.d
+      -  /static:/static
+    depends_on:
+      - web
+    networks:
+      - djangonetwork
   web:
     build: .
     container_name: dg01
-    command: bash -c "python manage.py makemigrations && python manage.py migrate && gunicorn my_django.wsgi -b 0.0.0.0:8000"
+    command: bash -c "sleep 7; python manage.py makemigrations && python manage.py migrate && gunicorn my_django.wsgi -b 0.0.0.0:8000"
```

Comment:
hmm, why adding `sleep 7` here?

Reply:
Yes, I figured out we can set a 'waitfor' so that should do just fine.
```Bash
environment:
  - WAIT_HOSTS: postgres:5432
```